language/Syntax.md

```diff
@@ -97,6 +97,13 @@ multiply x y = x * y
 
 Type signatures are not required for top-level declarations in general, but is good practice to do so. See the section on types for more details.
 
+## Function names
+
+Function names must start with a lowercase letter ([unicode category](https://en.wikipedia.org/wiki/Unicode_character_property#General_Category) `Ll`) and may be followed by any number of:
```

Comment:
I've realised that I missed a couple of cases in my description before: in addition to what you've got here, value names may also start with an underscore, and may also be followed by underscore or apostrophe (U+0027) `'`.